static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.jsx

```diff
@@ -100,19 +100,19 @@ describe('TemporalSelectionDropdown component', () => {
   test('on load should be closed on initial render', () => {
     setup()
 
-    expect(screen.queryByRole('button')).toBeInTheDocument()
+    expect(screen.getByRole('button')).toBeInTheDocument()
     expect(screen.queryByLabelText('Start')).not.toBeInTheDocument()
   })
 
   test('when clicked toggles the state of show ', async () => {
-    const user = userEvent.setup()
+    const { click } = userEvent.setup()
```

Comment:
Not sure why this change was made. But one thing we've done in other repos is call `userEvent.setup()` first in the `setup` function, then return the `user` object. So the test ends up looking like this `const { user } = setup({})`. That avoids having `userEvent.setup()` called 18 times in this file

Reply:
Hrmm, when I ran the linter it complained that this needed destructuring, but now it doesn't. I will change them back.

Reply:
Ahh, got it to complain again. 

```/Users/dpzamora/lab/cmr/earthdata-search/static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.jsx
  108:11  error  `user` is not a recommended name for `render` returned value. Instead, you should destructure it, or name it using one of: `view`, or `utils`  testing-library/render-result-naming-convention````

Reply:
I'll try it your way and see what happens.